src/components/experiment-tracking/run-export-modal/run-export-modal.js

```diff
@@ -15,13 +16,24 @@ const RunExportModal = ({
   setShowRunExportModal,
   theme,
   visible,
+  runsMetaData,
 }) => {
   const [exportData, setExportData] = useState([]);
   const { isSuccessful, showModal, handleClick } =
     useContext(ButtonTimeoutContext);
 
   const updateExportData = useCallback(() => {
-    setExportData(constructExportData(runMetadata, runTrackingData));
+    const mergedRunsMetadata = runMetadata?.map((run) => {
+      return {
+        ...run,
+        ...{
+          title: runsMetaData[run.id]?.title || run.id,
+          note: runsMetaData[run.id]?.title || '',
```

Comment:
are the keys in the run metadata - **title** and **notes** . I am not sure if I understand this clearly. Is the **note** key corresponds to different metadata ?

Reply:
Corrected, it was typo. Thanks